recipes-utils/cg5317-utils/files/25-seth0.network

```diff
@@ -0,0 +1,10 @@
+[Match]
+Name=seth0
+
+[Network]
+DHCP=yes
+Address=169.254.200.130/16
```

Comment:
any reason why we hard-code an IP here when DHCP is enabled?

Reply:
That is a link-local address as fallback. I did this because the .network in the v4.05 SDK had it (but with 169.250.x.x, which is not link-local).

Reply:
Potentially you are right and this is not necessary (or `LinkLocalAddressing=ipv4` may do the same) and I was misled because the SDK used the x.250.x.x to prevent the link from being marked as degraded.